CONTRIBUTING.md

```diff
@@ -0,0 +1,412 @@
+# Contributing to knative/docs
+
+- [Before you begin](#before-you-begin)
+- [Contributing to the documentation](#contributing-to-the-documentation)
+- [Docs contributor roles](#docs-contributor-roles)
+
+**First off, thanks for taking the time to contribute!**
+
+The following are guidelines for contributing to the Knative documentation.
+These are just guidelines, not rules. Use your best judgment, and feel free to
+propose changes to this document in a pull request.
+
+## Before you begin
+
+### Code of conduct
+
+Knative follows the [Knative Code of Conduct](../CODE-OF-CONDUCT.md). By
+participating, you are expected to uphold this code. Please report unacceptable
+behavior to knative-code-of-conduct@googlegroups.com.
+
+### Contributor license agreements
+
+We'd love to accept your contributions! But before we can take them, you need to
+fill out the [Google CLA](https://cla.developers.google.com). Important: You
+must fill out the CLA with the same email address that you used to create your
+GitHub account.
+
+Once you are CLA'ed, we'll be able to accept your pull requests. This is
+necessary because you own the copyright to your changes, even after your
+contribution becomes part of this project. So this agreement simply gives us
+permission to use and redistribute your contributions as part of the project.
+
+Note: Your contributions are verified using the email address for which you
+use to sign the CLA. Therefore, 
+[setting your GitHub account to private](https://help.github.com/en/articles/setting-your-commit-email-address)
+is unsupported because all commits from private accounts are sent from the `noreply` 
```

Comment:
<!--whitespace-bot-->
Remove trailing whitespace:
```suggestion
is unsupported because all commits from private accounts are sent from the `noreply`
```